README.md

```diff
@@ -120,9 +142,37 @@ builder.ToTable("Claims", "audit"); //similar for audit.Covers
 Hint! EntityTypeBuilder has a "HasData" method:
 
 ```csharp
-builer.HasData(new List<ClaimStatusEntity>
-    new() { Id = 1, Name = "Submitted", Description = "The claim has been submitted and is..."},
-    //...
+builder.HasData(
```

Comment:
Let's trim it to three statuses - we don't need all five and it just adds more noise.
Check the content i am adding to maintain consistency:
https://github.com/InsuranceTechnologySolutions/boosterconf-ef-ninja/pull/4/files#